### PR TITLE
Finex Redox keymap

### PR DESCRIPTION
**Pull request by finex**
Hi, I've created a custom keymap with Colemak Mod DH-MK, MacOS layers and some minor macros. The keymap is suited for International layout with dead keys. Colemak Mod DH-MK looks like a good compromise when you have to type in both Italian and English languages.

### CODE REVIEW

File: keyboards/redox/keymaps/finex/rules.mk [hunk 1]
```diff
@@ -0,0 +1,7 @@
+RGBLIGHT_ENABLE = yes
+
+COMMAND_ENABLE = no
+
+ifndef QUANTUM_DIR
```
Comment (drashna) [inline keyboards/redox/keymaps/finex/rules.mk]: Could you remove this (ifndef/include/endif) block? 

It's not needed, and is a holdover from the old build system.

**Comment by drashna** [comment]
Thanks!

File: keyboards/redox/keymaps/finex/rules.mk (hunk 1 above)
Comment (finex) [reply keyboards/redox/keymaps/finex/rules.mk]: Of course!

Comment (finex) [reply keyboards/redox/keymaps/finex/rules.mk]: @drashna  Need I to create a new pull request?

Comment (drashna) [reply keyboards/redox/keymaps/finex/rules.mk]: Nope, Just create a commit on the branch you created the pull request with, and push it.  That will automatically update the PR.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**